### PR TITLE
Docs: Add warning on UUIDs not being available before 10.2.0

### LINKED ISSUE #90169: Add disclaimer that fixed role UUIDs might not match if using an old Grafana version
https://github.com/grafana/grafana/pull/89457#issuecomment-2186203062

Add a disclaimer that fixed roled UUIDs are only set for newer instances and no migration has happened for older instances

### PR DESCRIPTION
**Pull request by Jguer**
Fixes #90169

### CODE REVIEW

**Review by eleijonmarck — APPROVED** [review]

File: docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ The following tables list permissions associated with basic and fixed roles.
 The following table has the existing built-in fixed role definitions. Other fixed roles might be added by plugins installed in Grafana.
 The UUID presented here can be used as an identifier for [Terraform provisioning](../rbac-terraform-provisioning).
 
+{{% admonition type="caution" %}}
+These UUIDs might not be available if your instance was populated before Grafana v10.2.0. You can use the roles API to determine the UUIDs for the roles you need. You can learn more about this [here]({{< relref "../manage-rbac-roles" >}}).
+{{% /admonition %}}
```
[the remark this answers is not in the document]
Comment (Jguer) [reply docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md:41]: Thanks for the review @jdbaldry , it's definitely less ambiguous. I think we can keep it as you rephrased it

[the remark this answers is not in the document]
Comment (jdbaldry) [reply docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md]: Thanks for taking a look. So is it correct to say it's definitely not available for instances created before v10.2.0?

If so, then you will want something like:

These UUIDs won't be available if your instance was created before Grafana v10.2.0.

File: docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ The following tables list permissions associated with basic and fixed roles.
 The following table has the existing built-in fixed role definitions. Other fixed roles might be added by plugins installed in Grafana.
 The UUID presented here can be used as an identifier for [Terraform provisioning](../rbac-terraform-provisioning).
 
+{{< admonition type="caution" >}}
+These UUIDs might not be available if your instance was created before Grafana v10.2.0.
```
Comment (ryantxu) [inline docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md:40]:
```suggestion
These UIDs might not be available if your instance was created before Grafana v10.2.0.
```
the ids are not UUIDs... just unique withint org

Comment (gamab) [reply docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md:40]: I think they are unique across all orgs as the role itself is unique across all orgs.

**Review by gamab — APPROVED** [review]

File: docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md (hunk 2 above)
Comment (Jguer) [inline docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md:40]:
```suggestion
These UUIDs won't be available if your instance was created before Grafana v10.2.0.
```

**Review by jdbaldry — APPROVED** [review]
❤️ Doc copy looks great in terms of style. Thanks for working with me on this.

**Pull request merged**